bool var_bool
byte var_byte
int16 var_int16
uint64 var_uint64
float32 var_float32
float64 var_float64
string var_string
geometry_msgs/Pose var_pose
std_msgs/ByteMultiArray[2] var_struct_array
std_msgs/ByteMultiArray[] var_struct_varlen_array
